website/content/en/v0.33/concepts/nodepools.md

```diff
@@ -285,7 +285,11 @@ kubelet:
 
 Karpenter will automatically configure the system and kube reserved resource requests on the fly on your behalf. These requests are used to configure your node and to make scheduling decisions for your pods. If you have specific requirements or know that you will have additional capacity requirements, you can optionally override the `--system-reserved` configuration defaults with the `.spec.template.spec.kubelet.systemReserved` values and the `--kube-reserved` configuration defaults with the `.spec.template.spec.kubelet.kubeReserved` values.
 
-For more information on the default `--system-reserved` and `--kube-reserved` configuration refer to the [Kubelet Docs](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#kube-reserved)
+{{% alert title="Note" color="primary" %}}
+When computing the ephemeral storage available on an instance type, Karpenter considers these reserved resources.
```

Comment:
Should this section match the other sections?

Reply:
Yes, copy miss from a previous iteration.